include/extractor/guidance/turn_instruction.hpp

```diff
@@ -43,8 +43,14 @@ enum TurnType // at the moment we can support 32 turn types, without increasing
     NewName,                // no turn, but name changes
```

Comment:
Right Now Notification is Unused due to data not being exposed. I wouldn't worry about that one right now. The moment we get more data into the system, we have to open a discussion of what to expose.